### PR TITLE
Jek/fieldarray

### PR DESCRIPTION
**Pull request by jkozdon**

### CODE REVIEW

**Comment by jkozdon** [comment]
Still running CI on both CPU and GPU.

**Comment by jkozdon** [comment]
cc  @lcw

File: experiment/euler_gravity/gravitywave.jl [hunk 1]
```diff
@@ -1,5 +1,6 @@
 using Atum
 using Atum.EulerGravity
+using Bennu: fieldarray
```
Comment (mwarusz) [inline experiment/euler_gravity/gravitywave.jl:3]: I don't think these `using` statements are needed because `Atum` reexports all of Bennu.

**Comment by mwarusz** [comment]
Merged in #10 . Thanks for you work on this @lcw @jkozdon !

**Pull request closed without merging**